### PR TITLE
Add sort and filter to collection table

### LINKED ISSUE #133: Sorting by column on collection list
Allow the user to sort ascending / descending based on a column.

### LINKED ISSUE #134: Filter collection list by collection name
Allow a user to filter the collections in the collection list based on the collection name.

Client side is OK given the limited number of entries in the collection list.

### PR DESCRIPTION
**Pull request by aaron-collier**
Fixes #133 
Fixes #134 

Screen Shot 2019-09-10 at 10 18 20 AM

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -13,3 +13,13 @@
 
 import '../src/application.scss';
 import 'bootstrap/dist/js/bootstrap';
+import 'datatables.net';
+
+$(document).ready(function() {
```
Comment (jcoyne) [inline app/javascript/packs/application.js]: Can we use `document.addEventListener("DOMContentLoaded", function(){})` rather than jquery here?

**Comment by aaron-collier** [comment]
@justinlittman yeah, that was just the initial pass (and the standard filter fox has the label "search") - I'll update that before moving out of DRAFT.

File: app/javascript/packs/application.js (hunk 1 above)
Comment (aaron-collier) [reply app/javascript/packs/application.js]: Done.

**Pull request merged**